cmd/ccboc/cmd/commands.go

```diff
@@ -364,6 +367,97 @@ func request(method, endpoint string, buffer *bytes.Buffer) ([]byte, *errorRespo
 	return body, nil, nil
 }
 
+func getCalculationPhase() error {
+	app := tview.NewApplication()
+	table := tview.NewTable().
+		SetBorders(true)
+
+	for {
+		calculationBulk, err := getCalculationBulkData()
+		if err != nil {
+			logrus.WithError(err).Fatal()
+		}
+
+		populateTable(calculationBulk, table)
+
+		app.SetInputCapture(func(event *tcell.EventKey) *tcell.EventKey {
+			if event.Key() == tcell.KeyEsc { // to exit the app with ESC
+				app.Stop()
+				os.Exit(0)
+			}
+			if event.Key() == tcell.KeyCtrlC { // to exit the app with CTRL+C
+				app.Stop()
+				os.Exit(0)
+			}
+			return nil
+		})
+
+		go func() {
```

Comment:
This will wait for 3 seconds and it will stop the app. Is this what we want here?

Reply:
The `Run` function below enters a loop and is blocking everything until `app.Stop()` comes in. That is why we had to create the goroutine to start the timer - stop the app after 3s.

Reply:
You don't start any timer here. This go routine will start, wait for 3 seconds and then stop the app. What is the purpose of that?

Reply:
We want to fetch the data and redraw the table of the calculations every 3s~. The problem is that `Run` is a blocking function, so we created the goroutine to stop the application (to fetch and redraw data).

Reply:
This is a really hacky way how to stop the app and redraw. Can you check if the library has a built-in way to dynamically update data inside the app?